xml/System.Numerics/Complex.xml

```diff
@@ -1832,6 +1832,42 @@
         <altmember cref="F:System.Numerics.Complex.Zero" />
       </Docs>
     </Member>
+    <MemberGroup MemberName="op_Addition">
+      <Docs>
+        <summary>Adds a specified number to another specified number, where at least one of them is a complex number, and the other could be a double-precision real number.</summary>
+        <remarks>
+          <format type="text/markdown">
+  <![CDATA[
+
+## Remarks
+
+The <xref:System.Numerics.Complex.op_Addition%2A> operator allows performing addition operations that involve complex numbers. It enables code such as the following:
+
+[!code-csharp[System.Numerics.Complex.Add#3](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.numerics.complex.add/cs/add3.cs#3)]
+[!code-vb[System.Numerics.Complex.Add#3](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.numerics.complex.add/vb/add3.vb#3)]
+
+If the addition results in an overflow in either the real or imaginary component, the value of that component is either <xref:System.Double.PositiveInfinity?displayProperty=nameWithType> or <xref:System.Double.NegativeInfinity?displayProperty=nameWithType>.
+
+Languages that do not support custom operators can call the <xref:System.Numerics.Complex.Add%2A> equivalent group of methods instead.
+
+The <xref:System.Numerics.Complex.op_Addition%2A> operators that receive one double are more efficient than the operators that receive two Complex numbers:
```

Comment:
*Moving this comment to the correct location...*

@tannergooding @rpetrusha @Jozkee I added a table inside each overloaded operator/method remark indicating the efficiency gain for that different operation kind, as described here: dotnet/corefx#15927

Let me know what you think.

Reply:
@tannergooding so we noticed the original proposal was submitted by one person, but they stopped replying so the issue got assigned to someone else who eventually submitted the PR to fix it. This means that the original flop calculations in the proposal may not be accurate.

Reply:
I don't think we need to list exact flops improvement numbers etc. Just a small comment calling out this is an optimization over creating a new complex and calling the other overload.

Reply:
Okay no problem.